packages/viewer/src/interactions/interactionApi.ts

```diff
@@ -194,6 +203,49 @@ export class InteractionApi {
     });
   }
 
+  public async rotateCameraAtPoint(
+    delta: Point.Point,
+    point: Point.Point
+  ): Promise<void> {
```

Comment:
There's code duplication with `rotateCamera` that suggests `point` could just be an arg to the existing function.

Reply:
Yeah, I moved this to a separate method to prevent breaking changes to anyone using `rotateAroundAxis` currently. I could certainly update that to call `rotateAroundAxisAtPoint` with the lookat point at a minimum. Open to removing `rotateAroundAxisAtPoint` and just adding the arg to `rotateAroundAxis` though if that seems like the better option